### ISSUE TITLE
Standard form responses for reports

### ISSUE THREAD

**Issue by Hendikins** [Weasyl/weasyl#130]
(Originally filed by Fiz as Redmine issue 422)

Moderators already use boilerplate for the majority of notes sent to users - a list of said boilerplate is available on the staff wiki.

Including this in the form for closing reports "action taken" will significantly help with streamlining the process. This could be implemented as a "predefined action" in the same way we currently have them for suspensions (/modcontrol/suspenduser).

**Comment by weykent** [comment]
Didn't this get added once already?

**Comment by Hendikins** [comment]
No. We replaced the existing set of predefined actions for suspensions, but the report system has never had this capability.
